03_capabilities.html

```diff
@@ -1 +1,179 @@
-<!-- Section removed as it was describing specific local end implementations -->
\ No newline at end of file
+<section>
+  <h2>Capabilities</h2>
+  <p>
+    Webdriver <dfn>capabilities</dfn> allow the <a>local end</a> to specify what
+     features it requires the <a>remote end</a> to fulfil to be able to create a
+      <a>new session</a>
+  </p>
+  <h3>Processing Capabilities</h3>
+  <p>
+    <p>The <a>remote end steps</a> to <dfn>process capabilities</dfn> are:</p>
+    <ol>
+      <li><p>Let <var>server capabilities</var> be a JSON Object with the following entries:<p></li>
+        <ol>
```

Comment:
A bit superfluous to say “Let foo be an entry” when the outer step says “… with the following entries:”.  This can just be a list, like:

```html
<li><p><code>"browserName"</code>: Lower-cased name of the user agent.</p></li>
```